projects/llm-d/visualizations/llmd_inference/plotting/error_report.py

```
@@ -215,6 +427,93 @@ def do_plot(self, *args):
+            header.append(html.Br())
+
+        # Summary section with PROW artifacts links
+        header.append(html.H3("PROW Artifacts Summary"))
+        header.append(html.P("Quick access to PROW artifacts for all test entries organized by platform → model → load_shape:"))
+
+        # Filter out unwanted settings keys
+        FILTERED_SETTINGS_KEYS = {"owner", "llm-d", "platform", "model", "load_shape", "version"}
+
+        # Group entries hierarchically
+        hierarchy = {}
+        for i, entry in enumerate(entries):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

(B007)